app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class HomeController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
```

Comment:
Maybe a personal preference thing - is middleware better defined on the routes, rather than the controllers?

Reply:
This is the default home controller with laravel, I also prefer defining the middleware in routes, but this controller is going to change in the next PR when I work on the UI for changing passwords and the likes. Happy for me to leave it for then?

Reply:
Yeah sure